checkstyle-suppressions.xml

```diff
@@ -19,4 +19,12 @@
 
   <!-- Don't check for JavaDocPackage in samples or tests-->
   <suppress checks="JavadocPackage" files=".*[/\\](samples|test)[/\\].*\.java"/>
+
+  <!-- code model -->
+  <suppress checks="[ParameterName|MemberName]" files="com[/\\]azure[/\\]autorest[/\\]extension[/\\]base[/\\]model[/\\]codemodel[/\\].*.java"/>
+  <!-- class type -->
+  <suppress checks="[ConstantName|MethodName]" files="com[/\\]azure[/\\]autorest[/\\]model[/\\]clientmodel[/\\](ArrayType|ClassType|GenericType|PrimitiveType).java"/>
```

Comment:
Not change these, as it feel more nature to have `ClassType.HttpPipelinePolicy` than `ClassType.HTTP_PIPELINE_POLICY`